src/hb-ot-stat-table.hh

```diff
@@ -96,23 +109,22 @@ struct AxisValueFormat1
   }
 
   bool keep_axis_value (const hb_array_t<const StatAxisRecord> axis_records,
-                        const hb_hashmap_t<hb_tag_t, float> *user_axes_location) const
+                        const hb_hashmap_t<hb_tag_t, Triple> *user_axes_location) const
   {
     hb_tag_t axis_tag = get_axis_tag (axis_records);
     float axis_value = get_value ();
 
-    if (!user_axes_location->has (axis_tag) ||
-        fabsf(axis_value - user_axes_location->get (axis_tag)) < 0.001f)
-      return true;
+    if (axis_value_is_outside_axis_range (axis_tag, axis_value, user_axes_location))
```

Comment:
same here:
return !axis_value_is_outside_axis_range (axis_tag, axis_value, user_axes_location)

and there's a few more places later on in this file that can be simplified in the same way.